website/docs/guides/mf-time-spine.md

```diff
@@ -0,0 +1,258 @@
+
+1. Navigate to the `models/marts` directory.
+2. Add a new YAML file named `_models.yml` with the following content:
+
+<File name='models/marts/_models.yml'>
+
+```yaml
+models:
+  - name: time_spine_daily
+    description: A time spine with one row per day, ranging from 2000-01-01 to 2030-01-01.
+    time_spine:
+      standard_granularity_column: date_day  # The base column used for time joins
+    columns:
+      - name: date_day
+        description: The base date column for daily granularity
+        tests:
+          - not_null
+          - unique
+        granularity: day
+```
+</File>
+
+This time spine YAML file:
+- Defines `date_day` as the base column for daily granularity.
+- Adds [tests](/docs/build/tests) for uniqueness and non-null values.
+- Configures `time_spine` properties so MetricFlow can use the table
+
+### Using an existing dim_date table
+
+This optional approach reuses an existing table, saving you the effort of creating a new one. However if you created a time spine table from scratch, you can skip this section. 
+
+If your project already includes a `dim_date` or similar table, you can configure it as a time spine:
+
+1. Locate the existing table (`dim_date`).
+2. Update `_models.yml` file to configure it as a time spine:
+
+<File name='_models.yml'>
+
+```yaml
+models:
+  - name: dim_date
+    description: An existing date dimension table used as a time spine.
+    time_spine:
+      standard_granularity_column: date_day
+    columns:
+      - name: date_day
+        granularity: day
+```
+</File>
+
+This time spine YAML file configures the `time_spine` property so MetricFlow can use the table.
+
+## Run and test the time spine
+
+For the time spine table you created, let's run it and validate the output.
+
+1. Run the time spine model to create the table:
+   ```bash
+   dbt run --select time_spine_daily
+   ```
+
+2. Validate the output by querying the generated table:
+   ```sql
+   select * from your_project_schema.time_spine_daily
+   ```
+
+3. Check that the table:
+   - Contains one row per day.
+   - Covers the date range you want (5 years back to 30 days forward)
+
+4. (Optional) If you have [metrics](/docs/build/metrics) already defined in your project, you can query the table/metrics using [Semantic Layer commands](/docs/build/metricflow-commands) to validate the time spine. 
+   
+   Let's say you have a `revenue` metric defined. You can query the table/metrics using the following command:
+
+    ```bash
+    dbt sl query --metrics revenue --group-by metric_time
+    ```
+
+    This will output results similar to the following in the dbt Cloud IDE:
+    <Lightbox src="/img/quickstarts/dbt-cloud/validate-mf-timespine-output.jpg" title="Validate the time spine output in dbt Cloud IDE" />
+
+5. Double check that the results are correct and returning the expected data.
+
+## Add additional granularities (optional)
```

Comment:
i thought we could do this based on [this example](https://docs.getdbt.com/docs/build/metricflow-time-spine#creating-a-time-spine-table) but when i tested it, it only returned date_day.

Reply:
did you get this working? I would be happy to hop on a call and co-dev-test together if you can find overlapping time!